cdap-common/src/main/java/io/cdap/cdap/common/lang/jar/BundleJarUtil.java

```diff
@@ -191,6 +198,22 @@ public static File prepareClassLoaderFolder(Location jarLocation, File destinati
     return destinationFolder;
   }
 
+  private static void linkDirectories(File sourceDirectory, File targetDirectory) throws IOException {
+    File[] files = sourceDirectory.listFiles();
```

Comment:
Use `DirUtils.listFiles` to simplify the logic.

Reply:
done